py/picca/bal_tools.py

```diff
@@ -12,50 +12,59 @@
 
 from . import constants
 
-
-def read_bal(filename):  ##Based on read_dla from picca/py/picca/io.py
-    """Reads the BAL catalog from a fits file.
+def read_bal(filename,survey='DESI'):  ##Based on read_dla from picca/py/picca/io.py
+    """Copies just the BAL information from the catalog.
 
     Args:
         filename: str
-            Catalog of BALs
+            Catalog name
+        survey: str
+           'DESI' or 'EBOSS'
 
     Returns:
-        A dictionary with BAL information. Keys are the THING_ID
+        A dictionary with BAL information. Keys are the TARGETID
         associated with the BALs. Values are a tuple with its AI
         (*_CIV_450) and BI (*_CIV_2000) velocity.
 
     """
+    if survey == 'EBOSS':
+        id_name = 'THING_ID'
+        ext_name = 1
```

Comment:
given that the previous code used BALCAT here, maybe change to `extname='BALCAT'`. Or is this to allow both a BAL catalog and an eBOSS style drq with BALs included? (then maybe add a comment about this)